ai/pipelines/audio-to-text.mdx

```diff
@@ -107,6 +107,18 @@ The following system requirements are recommended for optimal performance:
 - [NVIDIA GPU](https://developer.nvidia.com/cuda-gpus) with **at least 12GB** of
   VRAM.
 
+## Recommended Pipeline Pricing
+
+<Note>We are planning to simplify the pricing in the future so orchestrators can set 
+one AI price per compute unit and have the system automatically scale based on the 
+model's compute requirements.</Note>
+
+The pricing for the `audio-to-text` pipeline is based on competitor pricing. However, 
+we strongly encourage orchestrators to set their own pricing based on their costs and 
+requirements. Setting a competitive price will help attract more jobs, as Gateways can 
+set their maximum price for a job. The currently recommended pricing for this pipeline 
+is `0.02e-6 USD` per **milliseconds** of input.
```

Comment:
Grammar is a bit off. 3 options:

```The recommended pricing for this pipeline 
is currently `0.02e-6 USD` per **milliseconds** of input.
```

```The recommended pricing for this pipeline 
is `0.02e-6 USD` per **milliseconds** of input.
```

```The current recommended pricing for this pipeline 
is `0.02e-6 USD` per **milliseconds** of input.
```

Reply:
Good catch, thanks. Will make the corrections.